PluginGenerator/PluginDefinition.cs

```diff
@@ -38,13 +38,6 @@ public PluginDefinition()
         public string Organization { get; set; }
 
         /// <summary>
-        /// Returns additional files that should be added to the jar.
-        /// Key: relative path in jar
-        /// Value: the full path to the source file
-        /// </summary>
-        public IDictionary<string, string> AdditionalFileMap { get { return this.relativePathToFileMap; } }
```

Comment:
No longer required. The caller can now directly add the necessary files to the PluginBuilder.

This class was originally just a list of the properties that can be specified in the manifest file that will be picked up the SonarQube server and displayed in the UI. Over time a few additional properties were added. In fact, the only other non-manifest property left is *Extensions*, which can be removed too.